CONTRIBUTING.md

```diff
@@ -33,6 +33,15 @@ yarn pack
 You can then add this .tgz file with pnpm, or refer to it as follows:
 `"gosling.js": "file:<relative path to file>"`
 
+## Releasing a new version
+
+GitHub Action handles bumping the version of AltGosling. The following commands will automatically increase the version number and release the new version in the NPM server.
+
+```sh
+pnpm version --patch # or --minor or --major
+git push origin master --tags
```

Comment:
Does this mean someone needs to push directly to master? Or any branch with the tag that gets merged with master will do so? (In that case, perhaps we can clarify this)

Reply:
Is that a safety concern? Anyone with access to create a branch on the repo can release a new version. Can we restrain this to the master branch somehow?

Reply:
See my last comment below